### PR TITLE
README.md - RAM requirement on 32-bit linux

### PR DESCRIPTION
**Pull request by MagaTailor**
Running `/usr/bin/time -v make` to build rust (using local llvm) shows the maximum memory usage at 715 megabytes. (on arm linux it's even less @ 580M)

### CODE REVIEW

**Comment by brson** [comment]
Thanks for updating this.

Instead, can we maybe rephrase this entire sentence to something like 'Rust needs between 600MiB and 1.5GiB to build, depending on platform. If it hits swap...'?

Please also [squash](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) these commits.

**Comment by petevine** [comment]
Yeah, much cleaner! 
I'd already deleted my repo - do I have to resubmit or can you edit the PR?

**Comment by tshepang** [comment]
@petevine re-submit and close this one
